module1/homework/lengthSort/sort.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <deque>
+#include <forward_list>
+#include <string>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string>);
+bool CompareStrings(std::string, std::string);
```

Comment:
You declare function, which doesn't have definition.

Reply:
it should be removed